pkg/userdata/userdata.go

```diff
@@ -215,11 +218,14 @@ coreos:
       Environment="RKT_OPTS=\
         --uuid-file-save=/var/run/kubelet-pod.uuid \
         --volume etc-resolv,kind=host,source=/etc/resolv.conf --mount volume=etc-resolv,target=/etc/resolv.conf \
+        --volume etc-cni,kind=host,source=/etc/cni --mount volume=etc-cni,target=/etc/cni \
+        --volume opt-cni,kind=host,source=/opt/cni,readOnly=true --mount volume=opt-cni,target=/opt/weave-net \
```

Comment:
`DefaultNetworkProvider = "canal"` says it's `canal`, but this line says it's `weave`?